### ISSUE TITLE
Update footer to reflect GW-wide change

### ISSUE THREAD

**Issue by maxturer** [gwu-libraries/scholarspace-hyrax#560]
GW marketing & communications sent over the new accessibility footer structure and campaign links, GWSS should be updated to reflect
